### ISSUE TITLE
gsutil downloading and uploading issue

### ISSUE THREAD

**Issue by house-of-vanity** [thelastpickle/cassandra-medusa#167]
Hello, i've ran into an issue with medusa when backing up and restoring cluster. Medusa can't upload and download sstables using `gsutil cp` with error like this:
```
[2020-06-22 13:59:25,229] INFO: Downloading backup data
[2020-06-22 13:59:25,675] DEBUG: gsutil -m cp -c -L /tmp/gsutil_4a81a277-9b7c-4783-91db-a58e716c7b0a.manifest -I /tmp/medusa-restore-1f66c576-3308-4148-a9a5-e2e7015c81ad/ksp/segments-a95f7630800c11eaa121f94104d
b90c8
[2020-06-22 13:59:26,611] INFO: Downloading backup data
[2020-06-22 13:59:27,056] DEBUG: gsutil -m cp -c -L /tmp/gsutil_aa537884-8eb3-4ccc-8d8d-b169644fbb02.manifest -I /tmp/medusa-restore-1f66c576-3308-4148-a9a5-e2e7015c81ad/ksp/pe_bid-cf08ef60887211eaa121f94104db90c
8
[2020-06-22 13:59:28,030] INFO: Downloading backup data
[2020-06-22 13:59:28,485] DEBUG: gsutil -m cp -c -L /tmp/gsutil_3e7ff8ba-a1e2-43ac-b8ee-44f54389808c.manifest -I /tmp/medusa-restore-1f66c576-3308-4148-a9a5-e2e7015c81ad/ksp/pe_imp-cf9ddc60887211eaa121f94104db90c
8
[2020-06-22 14:01:25,838] DEBUG: Retrying (2/5)....
[2020-06-22 14:01:37,886] DEBUG: Retrying (3/5)....
...
```

After some investigation a found that keyspaces contains a lot of files affected this issue (~ 350 - 1200). I played with medusa code and fixed it in this PR. Feel free to change it somehow or ask me to update it.
https://github.com/thelastpickle/cassandra-medusa/pull/166

**Comment by arodrime** [comment]
@house-of-vanity / @adejanovski it seems #166 was merged. Should we close this issue?

Thanks @house-of-vanity for the report, and even more for participating suggesting a PR!

**Comment by dmitry-saprykin** [comment]
@adejanovski #166 changes seem to be absent from master. I saw the same issue with gsutil on master and it got fixed when I applied #166 changes.
